api-gateway-config/conf.d/management_apis.conf

```diff
@@ -35,16 +35,16 @@ server {
 
     location /v1/apis {
         access_by_lua_block {
-            local mgmt = require("management")
+            local apis = require("management/apis")
```

Comment:
WDYT if we move these lines inside the `management/apis` module and keep the nginx conf at the minimum with the amount of lua code. I.e.

```nginx
location /v1/apis {
  content_by_lua_block {
    local apis = require("management/apis")
    apis.requestHandler()
  }
}
```